.github/workflows/test.yml

```diff
@@ -10,33 +10,54 @@ on:
 
 jobs:
 
-  linux-x64:
-    runs-on: ubuntu-latest
+  test:
+    name: "Test ${{ matrix.os }}"
+    strategy:
+      fail-fast: false
+      max-parallel: 3
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+
+    runs-on: ${{ matrix.os }}
 
     steps:
+    - name: '[Windows] Install Cygwin'
+      if: runner.os == 'Windows'
+      uses: cygwin/cygwin-install-action@v2
+      with:
+        packages: >-
+          rsync
```

Comment:
You should not need rsync, with the latest version of the repo

Reply:
Removed `rsync` from the list of packages.